src/components/qna-components/utility/qna-utility.component.jsx

```diff
@@ -15,34 +17,57 @@ class Utility extends React.Component {
     this.setState({ questionHelpfulness });
   }
 
-  updateHelpful = () => {
+  updateHelpful = (e) => {
+    const { questionId } = this.props;
+    e.preventDefault();
     this.setState((previousState) => ({
       questionHelpfulness: previousState.questionHelpfulness + 1,
-    }));
+    }),
+    () => {
+      const { questionHelpfulClicked } = this.state;
+      this.setState({ questionHelpfulClicked: !questionHelpfulClicked }, () => {
+        fetch(`http://3.134.102.30/qa/question/${questionId}/helpful`,
+          { method: 'PUT' });
+      });
+    });
   }
 
+  // showAddAnswerModal = (e) => {
```

Comment:
Is there a good reason this is commented out?

Reply:
@danielmjordan this is in prep for my next step to build out the addAnswer popup modal, good question!